sda-commons-server-auth/src/main/java/org/sdase/commons/server/auth/key/JwksKeySource.java

```diff
@@ -156,6 +204,9 @@ private static class Key {
     private String use;
     private String n;
     private String e;
+    private String x;
+    private String y;
+    private String crv;
```

Comment:
Are these common/good names for EC?

Reply:
https://datatracker.ietf.org/doc/html/rfc7517#section-3 that's how it's provided